app/bundles/CoreBundle/Config/config.php

```diff
@@ -1177,7 +1181,6 @@
         'db_user'                         => '',
         'db_password'                     => '',
         'db_table_prefix'                 => '',
-        'db_server_version'               => '5.7',
```

Comment:
This prevents the `db_server_version` from being written to `app/config/local.php` moving forward 🚀